src/jdk.jdi/share/classes/com/sun/tools/example/debug/tty/EventHandler.java

```diff
@@ -96,6 +100,35 @@ public void run() {
     private boolean handleEvent(Event event) {
         notifier.receivedEvent(event);
 
+        /*
+         * See if the event thread is a vthread that we need to start tracking.
+         */
+        ThreadReference eventThread = null;
+        if (event instanceof ClassPrepareEvent) {
+            eventThread = ((ClassPrepareEvent)event).thread();
```

Comment:
```suggestion
        if (event instanceof ClassPrepareEvent evt) {
            eventThread = evt.thread();
```
The same for lines 109-110

Reply:
op